src/views/document/RouteView.vue

```diff
@@ -209,17 +201,15 @@ export default {
     },
   },
 };
-</script>
-
-var historyWorthActivities = [
+const historyWorthActivities = [
         'snow_ice_mixed',
         'mountain_climbing',
         'rock_climbing',
         'ice_climbing',
         'via_ferrata',
         'slacklining',
       ];
```

Comment:
Oh, didn't get the misplace `</script>` tag 🙁 
I assume I wasn't clear enough: the `historyWorthActivities` declaration should happend before the default export, i.e. before line 128.

Also, could you check the formatting and run prettier? The indentation is false.

Other than that, it's ok for me to merge 👍